opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampling_strategy.rs

```diff
@@ -107,9 +107,10 @@ impl Inner {
                 }
             })
             .unwrap_or_else(|_err| {
-                global::handle_error(TraceError::Other(
-                    "jaeger remote sampler mutex poisoned".into(),
-                ))
+                otel_warn!(
+                    name: "JaegerRemoteSampler.MutexPoisoned",
+                    message = "Unable to update Jaeger Remote sampling strategy: the sampler's internal mutex is poisoned, likely due to a panic in another thread holding the lock. The last known configuration will continue to be used until the remote sampling client is restarted.",
```

Comment:
what does it mean by "restarting remote sampling client" ? is that something user can trigger somehow?

Reply:
This was added based on the comment [here](https://github.com/open-telemetry/opentelemetry-rust/pull/2257#discussion_r1818371277). Remote sampling client is not a standalone entity, but part of otel-sdk. We can instead say that `the last known configuration will continue to be used until the remote sampling client is restarted` cc @TommyCpp

Reply:
if there is no way to restart sampling client independently, then lets state that the last known config will be used from now onwards/until process restarts.